lib/bunny/channel.rb

```diff
@@ -165,7 +165,7 @@ class Channel
     # @param [Bunny::Session] connection AMQP 0.9.1 connection
     # @param [Integer] id Channel id, pass nil to make Bunny automatically allocate it
     # @param [Bunny::ConsumerWorkPool] work_pool Thread pool for delivery processing, by default of size 1
-    def initialize(connection = nil, id = nil, work_pool = ConsumerWorkPool.new(1))
+    def initialize(connection = nil, id = nil, work_pool = ConsumerWorkPool.new(1,10))
```

Comment:
sorry to nitpick but please add a space between arguments, like we do everywhere else